docker/install-scrypted-dependencies-mac.sh

```diff
@@ -30,15 +30,54 @@ RUN pip3 install aiofiles debugpy typing_extensions typing opencv-python
 echo "Installing Scrypted..."
 RUN npx -y scrypted install-server
 
+RUN mkdir -p ~/Library/LaunchAgents
+RUN cat << EOF | tee ~/Library/LaunchAgents/com.scrypted.server.plist
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
+<plist version="1.0">
+<dict>
+    <key>RunAtLoad</key>
+        <true/>
+    <key>KeepAlive</key>
+        <true/>
+    <key>Label</key>
+        <string>com.scrypted.server</string>
+    <key>ProgramArguments</key>
+        <array>
+             <string>$(brew --prefix)/bin/npx</string>
```

Comment:
would use $(which npx) I think, or $0? Probably the prior. Should probably also do all sub shells first, with the RUN, to guarantee they succeed before creating a potentially busted launchd.